spec/factories/user.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
+    password { 'password' }
+    password_confirmation { 'password' }
```

Comment:
you can use Faker for password and repeat it in confirmation doing
`password_confirmation { password }`